index.html

```diff
@@ -158,64 +158,65 @@ <h1 id="title">Dataset Exchange Working Group Charter</h1>
 
-        <p>Sharing data among researchers, governments and citizens, whether openly or not, requires the provision of metadata. Different communities use different metadata standards to describe their datasets, some of which are highly specialized. At a general level W3C’s Data Catalog Vocabulary, <a href="https://www.w3.org/TR/vocab-dcat/">DCAT</a>, is in widespread use, but so too are <a href="https://ckan.org/">CKAN</a>’s native schema, schema.org's <a href="http://schema.org/Dataset">dataset description</a> vocabulary, <a href="http://www.iso.org/iso/catalogue_detail?csnumber=26020">ISO 19115</a>, <a href="http://www.ddialliance.org/explore-documentation">DDI</a>, <a href="https://sdmx.org/">SDMX</a>, <a href="http://www.eurocris.org/cerif/main-features-cerif">CERIF</a>, <a href="https://www.w3.org/TR/void/" title="Describing Linked Datasets with the VoID Vocabulary">VoID</a>, <a href="http://inspire.ec.europa.eu/index.cfm/pageid/101">INSPIRE</a> and, in the healthcare and life sciences domain, the <a href="https://www.w3.org/TR/hcls-dataset/">Dataset Description vocabulary</a> and <a href="https://github.com/biocaddie/WG3-MetadataSpecifications">DATS</a> (<a href="https://doi.org/10.1101/103143" title="DATS: the data tag suite to enable discoverability of datasets">ref</a>) among others.  This variety is a clear indication that no single vocabulary offers a complete and universally accepted solution. </p>
+        <p>A success of a previous WG charter was the publication of DCAT-3, where the evolved through multiple drafts to improve clarity, extend functionality, and align with best practices by introducing dataset series as first-class entities, refining versioning and inverse property handling, enhancing multilingual and accessibility support, improving alignment with external standards, and strengthening guidance on security, privacy, and metadata usage. </p>
```

Comment:
I have troubles parsing the beginning of the first sentence. I'm guessing that "where the" can be read as "which" (see below)? Also I find this sentence very long, so I suggest to split it in two:

```suggestion
        <p>A success of a previous WG charter was the publication of DCAT-3, which evolved through multiple drafts to improve clarity, extend functionality, and align with best practices. In particular, it introduced dataset series as first-class entities, refined versioning and inverse property handling, enhanced multilingual and accessibility support, improved alignment with external standards, and strengthened guidance on security, privacy, and metadata usage. </p>
```